frontend/src/screens/dashboard/docsqa/index.tsx

```diff
@@ -274,11 +249,74 @@ const DocsQA = () => {
                 onChange={(e) => setPromptTemplate(e.target.value)}
               />
             </div>
+            <div className="h-full border rounded-lg border-[#CEE0F8] w-[calc(100%-400px)] bg-white p-4">
+              <div className="flex gap-4 items-center">
+                <div className="w-full relative">
+                  <Input
+                    className="w-[100%] h-[44px] text-sm pr-14"
```

Comment:
- `w-[100%]` is not needed as `w-full` is already defined in tailwind
- Please don't use pixel values, only use rems